bundler/script/ci-test

```diff
@@ -4,11 +4,12 @@ set -e
 
 bundle install
 bundle exec rubocop .
-bundle exec rspec spec
+bundle exec parallel_test spec/ -n "$CI_NODE_TOTAL" --only-group "$CI_NODE_INDEX" --group-by filesize --type rspec
 
 # NOTE: Don't use `if` branches without `else` part, since the code in some of
 # then seems to not abort the script regardless of `set -e`
 
+# Should we only run these on one of the CI_NODE_INDEX's?
```

Comment:
Is this a future TODO that should be merged in or does it need addressing/removing before merge?

Reply:
I was treating it as a future TODO. Should be easy to tackle, but I don't really have more spare cycles to burn on this